js/pages/home.js

```diff
@@ -1,8 +1,7 @@
 // component imports
 
-import { renderPlans } from '../components/plansList/plansList.js';
-import { plansData } from '../data/plansData.js';
-
+import renderLatestpost from '../components/latestPost/latestPostList.js';
+import latestPostData from '../data/latestPostData.js';
 
```

Comment:
sukurk versija kur butu visi importai

Reply:
ta prasme ideti developo importa i savo home.js?

Reply:
gal uztektu tik prideti atitinkamai tusciu eiluciu ?

Reply:
neina, formatavimas istrina :D gal tiesiog copy paste is to kas jau padaryta pas Dovlie ir Kaspara. Let me know jei veikia, as ir taip pasidarysiu :D

Reply:
tai kad as nesuzinosiu ar isejo kazkas jei nieks neaprovins